database/migrations/2020_01_18_141915_add_participant_information_channel.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddParticipantInformationChannel extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('participants', function (Blueprint $table) {
+            $table->enum('information_channel', ["postal and email", "only email"])
```

Comment:
Prima naam voor de kolom, ben alleen een beetje frowny op de spaties in de enums, ik weet dat het hier strings zijn en dat het kan maar in andere talen mag het niet. Maar goed dat is misschien niet zo'n sterk argument

Reply:
In database land is het heel arbitrair, want stiekem is het gewoon of een kleine lookup  table van een text naar een integer of numeric (bij postgres is het een numeric, want je kan je enum altere en er halfwege een enum value bij vrotten... echt top, staat er ineens een enum waarde van 3.5 in je tabel). Maar ik snap je argument. Ik verander het naar iets met streepjes, dan is het duidelijker dat het geen user-facing text is.